extensions/pvaPublish/tsconfig.json

```diff
@@ -1,7 +1,8 @@
 {
```

Comment:
Nit: Should you be importing a base tsconfig that has the stricter settings?

Reply:
I have another PR incoming for this package that deals with types, I will add some of the strict settings in that PR.